src/components/map/WebViewMap.tsx

```diff
@@ -21,7 +21,7 @@ export const WebViewMap = ({
   mapCenterPosition,
   onMessageReceived,
   style,
-  zoom
+  zoom = 10
```

Comment:
how come 10?

example with that value:

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-03 at 18 39 50](https://user-images.githubusercontent.com/1942953/109847999-17980a00-7c50-11eb-8250-29be5910b47e.png)

when i am using the bad belzig app, it hould be clear, that the location is somewhere there. so a more close view could be useful, maybe:

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-03 at 18 40 20](https://user-images.githubusercontent.com/1942953/109848169-3f876d80-7c50-11eb-8e07-17986adbf6b4.png)

or something in between?

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-03 at 18 44 25](https://user-images.githubusercontent.com/1942953/109848395-81181880-7c50-11eb-867f-756766d54da2.png)

Reply:
increased it to 14 for POIs and construction sites